src/Cli/ConfigGenerator.cs

```diff
@@ -143,13 +143,19 @@ public static bool TryCreateRuntimeConfig(InitOptions options, FileSystemRuntime
                     string? cosmosDatabase = options.CosmosNoSqlDatabase;
                     string? cosmosContainer = options.CosmosNoSqlContainer;
                     string? graphQLSchemaPath = options.GraphQLSchemaPath;
-                    if (string.IsNullOrEmpty(cosmosDatabase) || string.IsNullOrEmpty(graphQLSchemaPath))
+                    if (string.IsNullOrEmpty(cosmosDatabase))
                     {
-                        _logger.LogError("Missing mandatory configuration option for CosmosDB_NoSql: --cosmosdb_nosql-database, and --graphql-schema");
+                        _logger.LogError("Missing mandatory configuration options for CosmosDB_NoSql: --cosmosdb_nosql-database, and --graphql-schema");
                         return false;
                     }
 
-                    if (!fileSystem.File.Exists(graphQLSchemaPath))
+                    if (string.IsNullOrEmpty(graphQLSchemaPath))
+                    {
+                        graphQLSchemaPath = "schema.gql"; // Default to schema.gql
```

Comment:
what if the default file itself is not found in the directory or is inaccessible?

Reply:
it is not looking for the file, it will create a file with the name.

Reply:
And if the file already exists? will it use the one already present or will it overwrite the existing file?